tracer/samples/Samples.Transport.UnixDomainSocket/Samples.Transport.UnixDomainSocket.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UserSecretsId>f9c4f19d-b9f0-4b05-8374-a3fa4972949b</UserSecretsId>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Remove optimizations and always use Dockerfile -->
+    <ContainerDevelopmentMode>Regular</ContainerDevelopmentMode>
+    <!-- Build at solution level to allow referencing the tracer -->
+    <DockerfileContext>..\..\..</DockerfileContext>
+    <!-- Pass any sensitive information as arguments -->
+    <DockerfileBuildArguments>--build-arg DD_API_KEY=$(DD_API_KEY)</DockerfileBuildArguments>
```

Comment:
Do you know if --build-arg complains about an empty setting or if the Dockerfile already complains about an empty ARG? I want to make sure that if $(DD_API_KEY) is not set, the user gets a clear error instead of trying to figure out why the app is running and not sending traces/metrics (when using the real Datadog agent)

Reply:
The build output fails at the agent install.. Is that good enough or do you think we should do more?
![image](https://user-images.githubusercontent.com/1801443/148441751-583240a4-f33f-4fbe-8019-4e62b4d55fb3.png)

Reply:
I think that's good enough 😄